### ISSUE TITLE
Xenos can't open their eggs by clicking on them. Also other xeno related bugs

### ISSUE THREAD

**Issue by Shadowlight213** [tgstation/tgstation#20997]
They instead attack them.
The alien queen smashes into the egg!

@phil235

**Comment by Shadowlight213** [comment]
Xeno corrosive acid is also broke

**Comment by Shadowlight213** [comment]
Also xenos can't unsecure someone from a nest. Trying to will cause them to just attack it until it is destroyed.
Intent doesn't matter.

**Comment by FantasticFwoosh** [comment]
Xeno eggs never have opened on touch, that is a CM feature, on TG typically they open when they are ready, this is to prevent premature dangerous facehugger spam, our facehuggers are more active than theirs that sit quietly, and ours open on a 'timer' because they are VERY powerful already.

Secondly all of those additional issues really deserve their own issue reports separately
> Xeno corrosive acid takes time, there is a message that it is working in the ic chat bar, at the start that can determine if it'll take a while to acid through.
> The last one sounds more like a actual issue, though you will need to target SPECIFICALLY on help intent and not hit the surrounding weeds/mobs encased in the nest.

**Comment by PKPenguin321** [comment]
>Xeno eggs never have opened on touch

Objectively wrong, play moar

**Comment by phil235** [comment]
I didn't see the second bug. Will fix too.

**Comment by RemieRichards** [comment]
@FantasticFwoosh you are quite literally as wrong as you could be right now.
Xeno eggs have opened on touch for about as long as we've had xeno eggs.

The fact CM has the feature is because they're a fork of ancient-bay, and ancient-bay ALSO had the same Xeno egg mechanics as us (almost certainly the exact same code too)

tl;dr you're wrong, stop digging this hole.
